support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerListener.java

```diff
@@ -29,4 +35,24 @@ public void onDeleted(Run<?,?> run) {
         }
     }
 
+    @Extension
+    public static final class Listener extends RunListener<Run<?, ?>> {
```

Comment:
Why a separate listener? You can just put `onStarted` directly in `BuildTriggerListener`.